src/View/Admin/WooTab.php

```diff
@@ -153,7 +160,11 @@ public function __construct() {
 	 */
 	public function register_hooks() {
 		add_filter( 'woocommerce_settings_tabs_array', [ $this, 'add_settings_page' ], 99 );
+		add_action( "woocommerce_settings_cc_woo_store_information_settings_data", [ $this, 'add_optional_fields_wrapper' ] );
+		add_action( "woocommerce_settings_cc_woo_store_information_settings_data_end", [ $this, 'add_optional_fields_wrapper_end' ] );
+
 		add_action( "woocommerce_settings_{$this->id}", [ $this, 'output' ] );
+		// add_action( "woocommerce_settings_tabs_{$this->id}", [ $this, 'override_save_button' ] );
```

Comment:
Is this not needed in the end? or just temporarily not needed, and may be in the future?